build.gradle.kts

```diff
@@ -27,6 +27,7 @@ plugins {
     id("com.android.application") apply false
     id("com.avito.android.instrumentation-tests") apply false
     id("com.avito.android.impact")
+    // if(ci) id("com.avito.android.build-verdict")
```

Comment:
you should not conditionally apply plugin; it should be inside plugin's logic

Reply:
Let's discuss this. For me is a clear intention I like a user want to decide when I want to apply plugin

Reply:
it's kinda gradle's design decision

https://discuss.gradle.org/t/how-to-apply-plugin-based-on-build-type-android/15265/2?u=jffiorillo

some part of script logic depends on it; like kotlin dsl won't compile if you're using generated accessors

Reply:
it's not the better citation i found, not really about this case; but i seen better somewhere